### DISCUSSION TITLE
Allow querying multiple entries by version at once

### DISCUSSION THREAD

**Discussion by TECHNOFAB11** [directus/directus#24090] [Feature Requests]
### Describe the Improvement

As far as I can see, currently content versions can only be used for single items (like with a given ID or Singletons). Which makes sense as versions are item specific. 

But the use case I imagine could look like this:

Staging version of a site should be used to preview changes before promoting them to Main and thus production. 
The staging site thus asks for the "staging" version of all the data. If it doesn't exist just fallback to Main like Directus already does automatically. 
But instead of having to query multiple times (once to get the items' ids and then query the items individually with a version specified) it would be great to be able to just have a filter which works with versions as well. So I can for example query all blog posts for the overview page, all with version "staging" to be able to preview changed titles for example. Better example would just be some sections on a landing Page, currently every section would need to be queried individually, better would be a single query.

**Comment by rijkvanzanten** [comment]
Heya! Thanks for your suggestion! 

As this is a bit of a bigger project, I'll move it to feature requests instead (we use issues for individual actionable items) 🙂
